zookeeper-server/src/test/java/org/apache/zookeeper/server/ToStringTest.java

```diff
@@ -37,4 +41,18 @@ public void testJuteToString() {
         assertNotSame("ERROR", req.toString());
     }
 
+    @Test
+    public void testOpCodeToString() throws Exception {
+        Class<?> clazz = ZooDefs.OpCode.class;
+        Field[] fields = clazz.getFields();
+
+        assertNotEquals(0, fields.length);
+
+        for (Field field : fields) {
+            int op_code = field.getInt(null);
```

Comment:
1. Camel-Case. e.g. op_code --> `opCode`
2. we should assert here `assertEquals(field.getName(), opString)`; ? (Of course we need to modify `createTTL`). When adding a new OpCode, this unit case can tell us: don't forget the method: `Request.op2String`

Reply:
I have pushed a fixup commit with changes requested.